patches/minecraft/net/minecraft/util/text/ITextComponent.java.patch

```diff
@@ -8,3 +8,19 @@
      String func_150254_d();
  
      List<ITextComponent> func_150253_a();
+@@ -83,6 +82,7 @@
+                     JsonObject jsonobject = p_deserialize_1_.getAsJsonObject();
+                     ITextComponent itextcomponent;
+ 
++                    if ((itextcomponent = net.minecraftforge.common.ForgeHooks.onTextComponentDeserialize(jsonobject)) != null) {/*NOOP*/} else 
```

Comment:
Well you're still doing that but under the `else` instead

I have seen in patch files that keeping the diff down is preferred over good code standards